src/subscription_manager/i18n.py

```diff
@@ -46,6 +48,7 @@ def configure_i18n():
     except locale.Error:
         locale.setlocale(locale.LC_ALL, 'C')
     configure_gettext()
+    Locale.set(os.environ.get("LANG"))
```

Comment:
Shouldn't this come before the configure call?

Reply:
I don't think so. This Locale is our locale class which uses gettext.

Reply:
ok